app/src/main/java/com/elmeradrianv/shesafe/database/EmergencyMessage.java

```diff
@@ -0,0 +1,28 @@
+package com.elmeradrianv.shesafe.database;
+
+import com.parse.ParseObject;
+
+import org.json.JSONArray;
+
+public class EmergencyMessage extends ParseObject {
+    public static final String MESSAGE_KEY = "message";
+    public static final String USER_KEY = "user";
+    public static final String EMERGENCY_CONTACTS_KEY = "emergencyContacts";
+
+    public String getMessage() {
```

Comment:
I don't think you're using this class yet. You can leave it in for now but in the future don't commit any code you aren't using.

Reply:
Yeah, that was my mistakes, it's supposed that be in the Database_model branch. But, I need to pull the code of the Parse DB in the signup branch and contain all the code (also, class that I don't use already)